Makefile

```diff
@@ -46,7 +46,7 @@ ASSISTED_SERVICE_DOCKER_DAEMON_PULL_STRING := $(or ${ASSISTED_SERVICE_DOCKER_DAE
 # We decided to have an option to change replicas count only while running in minikube
 # That line is checking if we run on minikube
 # check if SERVICE_REPLICAS_COUNT was set and if yes change default value to required one
-REPLICAS_COUNT = $(shell if ! [ "${TARGET}" = "minikube" ];then echo 3; else echo $(or ${SERVICE_REPLICAS_COUNT},3);fi)
+REPLICAS_COUNT = $(shell if ! [ "${TARGET}" = "minikube" ];then echo 1; else echo $(or ${SERVICE_REPLICAS_COUNT},3);fi)
```

Comment:
it means that we are changing the default for all deployments, not only day2. We must not interfere with other deployments. Lets leave default on 3, and always set SERVICE_REPLICAS_COUNT for our deplyoment

Reply:
yeah, that was just for testing :) done.